portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/ext/EnvironmentVariables.testcase

```diff
@@ -0,0 +1,117 @@
+@component-name = "portal-util-misc"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Ext";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Verify portal-ext properties can be set with environment variables after setting company-id-properties=true via jvm argument."
+	@priority = "3"
+	test ConfigurePortalExtPropertyViaJVM {
+		property test.name.skip.portal.instance = "EnvironmentVariables#ConfigurePortalExtPropertyViaJVM";
+
+		task ("Given a feature flag is disabled") {
+			ApplicationsMenuHelper.openApplicationsMenu();
+
+			AssertElementNotPresent(
+				key_category = "Import / Export",
+				key_portlet = "Import/Export Center",
+				locator1 = "ApplicationsMenu#PORTLET");
+
+			Refresh();
+		}
+
+		task ("When the feature flag is enabled via environment variable and company-id-properties=true is set via jvm argument") {
+			var liferayHome = PropsUtil.get("liferay.home.dir.name");
+
+			var setenvFile = "${liferayHome}/tomcat-9.0.65/bin/setenv.sh";
```

Comment:
This is technically broken now on master due to tomcat update, use ${app.server.tomcat.dir} instead (still have to grab the property)

Reply:
@kyle-miho How do I grab that property in Poshi? Every example I see of the tomcat directory being mentioned just hardcodes the directory. PropsUtil doesn't work.

Reply:
Check
<target name="prepare-poshi-runner-properties">
PropsUtil doesn't work because properties in Poshi are manually loaded by what is in poshi.properties or poshi-ext.properties i believe (Check Poshi's PropsUtil.java)

We can add the property to be loaded so we can call PropsUtil.java on it. the property should already exist, as app.server.bin.dir

Reply:
Let me know if you need more help on this part